message.go

```diff
@@ -9,14 +9,13 @@ import (
 	"path/filepath"
 )
 
-type Token struct {
-	URL       string `json:"URL"`
-	Buildkit  string `json:"Buildkit"`
-	Registry  string `json:"Registry"`
-	ID        string `json:"ID"`
-	Username  string `json:"Username"`
-	Token     string `json:"Token"`
-	MachineID string `json:"MachineID"`
+type contexts struct {
+	Current  string             `json:"current-context"`
+	Contexts map[string]context `json:"contexts"`
+}
+
+type context struct {
+	Name string `json:"name"`
```

Comment:
since this is the only thing we need, I figure it might be easier to just parse that.